code/control/DeployPlanDispatcher.php

```diff
@@ -178,6 +179,49 @@ protected function createFetch() {
 	}
 
 	/**
+	 * @param SS_HTTPRequest $request
+	 *
+	 * @return SS_HTTPResponse
+	 */
+	public function deploysummary(SS_HTTPRequest $request) {
+
```

Comment:
I'd suggest not adding newlines on the first line of functions if you can help it, it's kind of annoying (IMHO)